src/main/kotlin/spatialcrafting/compat/rei/ReiSpatialCraftingPlugin.kt

```diff
@@ -11,39 +15,32 @@ import spatialcrafting.recipe.SpatialRecipe
 import spatialcrafting.util.itemStack
 import java.util.function.Function
 
-class ReiSpatialCraftingPlugin : REIPluginEntry {
+class ReiSpatialCraftingPlugin : REIPluginV0 {
     companion object {
         val Id = modId("rei_plugin")
         private val CrafterSizes = SmallestCrafterSize..MaxCrafterSize
     }
 
+    override fun getMinimumVersion(): SemanticVersion = SemanticVersion.parse("3.0-pre")
 
     override fun getPluginIdentifier(): Identifier = Id
 
-    override fun onFirstLoad(pluginDisabler: PluginDisabler?) {
-        super.onFirstLoad(pluginDisabler)
-    }
-
-    override fun registerItems(itemRegistry: ItemRegistry?) {
-        super.registerItems(itemRegistry)
-    }
-
     override fun registerPluginCategories(recipeHelper: RecipeHelper) {
         for (i in CrafterSizes) {
             recipeHelper.registerCategory(ReiSpatialCraftingCategory(i))
         }
 
     }
 
-    private fun <T : Recipe<*>> RecipeHelper.registerRecipes(size: Int, recipeFilter: (Recipe<*>) -> Boolean, mappingFunction: (T) -> RecipeDisplay<*>) {
+    private fun <T : Recipe<*>> RecipeHelper.registerRecipes(size: Int, recipeFilter: (Recipe<*>) -> Boolean, mappingFunction: (T) -> RecipeDisplay) {
         registerRecipes(ReiSpatialCraftingCategory.id(size), Function(recipeFilter), Function(mappingFunction))
     }
 
 
     override fun registerRecipeDisplays(recipeHelper: RecipeHelper) {
         for (i in CrafterSizes) {
             recipeHelper.registerRecipes<SpatialRecipe>(i,
-                    recipeFilter = { recipe -> recipe is SpatialRecipe && recipe.minimumCrafterSize == i  },
```

Comment:
wow

Reply:
It is a reformatting, look at the line under this